addons/find-bar/userscript.js

```diff
@@ -285,7 +285,7 @@ export default async function ({ addon, msg, console }) {
         if (root.type === "event_whenbroadcastreceived") {
           const fieldRow = root.inputList[0].fieldRow;
           let eventName = fieldRow.find((input) => input.name === "BROADCAST_OPTION").getText();
-          addBlock("receive", "event " + eventName, root).eventName = eventName;
+          addBlock("receive", msg("event") + " " + eventName, root).eventName = eventName;
```

Comment:
I think you forgot to pass `{name: eventName}` here.

Reply:
Thanks!